src/App.jsx

```diff
@@ -6,13 +6,15 @@ import NotFound from "./pages/NotFound";
 import ReportPage from "./pages/ReportPage";
 
 import "./App.css";
+import AddReportPage from "./pages/AddReportPage";
 
 function App() {
   return (
     <Routes>
       <Route path="/" element={<Layout />}>
         <Route index element={<Home />} />
         <Route path="report" element={<ReportPage />} />
+        <Route path="addreport" element={<AddReportPage />} />
```

Comment:
we're opening a modal, looks like hindi na natin need ito

Reply:
i agree with this one, di na nagrreflect sa url yung path

Reply:
ay yeah I used that on my early testing lang